Source/Widgets/Viewer/Viewer.js

```diff
@@ -176,6 +180,55 @@ define([
         });
     }
 
+    function enableVRUI(viewer, enabled) {
+        var geocoder = viewer._geocoder;
+        var homeButton = viewer._homeButton;
+        var sceneModePicker = viewer._sceneModePicker;
+        var baseLayerPicker = viewer._baseLayerPicker;
+        var animation = viewer._animation;
+        var timeline = viewer._timeline;
+        var fullscreenButton = viewer._fullscreenButton;
+        var infoBox = viewer._infoBox;
+        var selectionIndicator = viewer._selectionIndicator;
+
+        var display = enabled ? 'none' : 'block';
```

Comment:
You should perhaps use `visibility` here instead of `display`.  The geocoder's proper display style is `inline-block`, not `block`, so this code causes problems with the toolbar.